### PR TITLE
Activate test

### PR DESCRIPTION
**Pull request by azuchi**
The test could not have been run by commit 2ad13a1f58a7e6b75d8f46513c260e251ba51d4c. So enabled this and fix a case where the test would fail because it wasn't mocked.

### CODE REVIEW

File: frontend/src/app/blocks/blocks.page.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class BlocksPage implements OnInit {
       data => {
         const resultData: any = data || {};
         const txnsData: any = resultData.results || [];
+        console.log(txnsData);
```
Comment (rantan) [inline frontend/src/app/blocks/blocks.page.ts:36]: This should be removed before marge.

File: backend/actions/transaction_list.js [hunk 2]
```diff
@@ -42,8 +42,8 @@ app.get('/transactions', async (req, res) => {
   const page = Number(req.query.page);
 
   try {
-    const chainTxStats = await tapyrusd.getChainTxStats();
-    const txCount = chainTxStats.txcount;
+    // const chainTxStats = await tapyrusd.getChainTxStats();
```
Comment (rantan) [inline backend/actions/transaction_list.js:45]: This should be removed. :scissors:

**Review by rantan — CHANGES_REQUESTED** [review]

File: frontend/src/app/blocks/blocks.page.ts (hunk 1 above)
Comment (azuchi) [reply frontend/src/app/blocks/blocks.page.ts]: fix it

File: backend/actions/transaction_list.js (hunk 2 above)
Comment (azuchi) [reply backend/actions/transaction_list.js]: fix it

**Review by rantan — APPROVED** [review]

**Pull request merged**